packages/composable-controller/src/ComposableController.test.ts

```diff
@@ -224,14 +224,14 @@ describe('ComposableController', () => {
           ensEntries: {},
         },
         NetworkController: {
+          networkConfigurations: {},
```

Comment:
These changes should no longer be needed as of this PR: https://github.com/MetaMask/core/pull/1077